src/builder.rs

```diff
@@ -341,6 +355,10 @@ impl SessionBuilder {
             init.arg("-o").arg(format!("Compression={}", arg));
         }
 
+        if let Some(ssh_auth_sock) = self.ssh_auth_sock.as_deref() {
+            init.env("SSH_AUTH_SOCK", ssh_auth_sock);
```

Comment:
Can this not be passed using an argument instead?

Reply:
I checked the man page of `ssh` and googled it but didn't find anything